apps/prairielearn/assets/scripts/instructorAssessmentInstancesClient.ts

```diff
@@ -412,6 +412,7 @@ onDocumentReady(() => {
         <select
           class="custom-select select-time-limit"
           name="plus_minus"
+          aria-label="time limit options"
```

Comment:
These labels will be displayed in e.g. the VoiceOver rotor, so it's best to use Sentence case (capitalize the first letter) just like one would with a visible form/button label. Ditto everywhere else in this PR.

Reply:
Updated here and elsewhere to Sentence case in [b2ddc80](https://github.com/PrairieLearn/PrairieLearn/pull/10514/commits/b2ddc80c9af2e30a4fa6ea94027150ce2ac8b3ba).